src/invidious/helpers/oauth.cr

```diff
@@ -0,0 +1,47 @@
+def oauth_get(idx)
```

Comment:
Coud you put everything in this file under the `Invidious::OAuthHelper` module, and remove the `oauth_` prefix from variables? (don't forget to `extend self` on the module ^^)